### PR TITLE
Add go_package annotations to all protos

### PR DESCRIPTION
**Pull request by keith**

### CODE REVIEW

File: xds/annotations/v3/status.proto [hunk 1]
```diff
@@ -4,6 +4,8 @@ package xds.annotations.v3;
 
 import "google/protobuf/descriptor.proto";
 
+option go_package = "github.com/cncf/xds/go/xds/annotations/v3";
```
Comment (keith) [inline xds/annotations/v3/status.proto:7]: I'm not actually sure if this should include the second xds or not, in the other case I omitted this, but can't recall why

**Comment by keith** [comment]
I guess at this point I should add these to all protos?

**Comment by htuch** [comment]
@keith if you can do them all and use consistent package namespace, SG.

**Comment by keith** [comment]
Ok I got them all here.

**Review by htuch — APPROVED** [review]

**Pull request merged**